### ISSUE TITLE
recursion limit when loading SimplicialComplexes package

### ISSUE THREAD

**Issue by d-torrance** [Macaulay2/M2#694]
I get the following error when attempting to load the SimplicialComplexes package:

    Macaulay2, version 1.10
    with packages: ConwayPolynomials, Elimination, IntegralClosure, InverseSystems,
                   LLLBases, PrimaryDecomposition, ReesAlgebra, TangentCone
    
    i1 : loadPackage "SimplicialComplexes"
    SimplicialComplexes.m2:15:1:(2):[297]: error: recursion limit of 300 exceeded

**Comment by DanGrayson** [comment]
Try adding`-q` to the command line.

**Comment by d-torrance** [comment]
Strange -- I tried running M2 from the command line so I could add `-q` and it worked without any problems, both with and without `-q`.

But it kept happening when I ran M2 from emacs.  I edited /usr/share/emacs/site-lisp/M2.el to add `-q` and everything worked.  Then I reverted it back to how it was and it works again!

So no idea what happened, but everything is fine.  Sorry for the noise!  Closing this and #693.
